src/svg-quirks-mode/svg-renderer.js

```diff
@@ -73,6 +73,13 @@ class SvgRenderer {
     }
 
     /**
+     * @return {[number,number]} the natural size, in Scratch units, of this SVG.
+     */
+    get size () {
```

Comment:
Open to suggestions for this name. It seems odd for a renderer to have a size, but I couldn't think of anything better to call it.